packages/plugins/src/plugins/glint-report.plugin.ts

```diff
@@ -83,9 +83,7 @@ export class GlintReportPlugin extends Plugin<GlintReportPluginOptions> {
     const program = languageService.getProgram()!;
     const checker = program.getTypeChecker();
 
-    const diagnostics = getDiagnosticOrder(service.getDiagnostics(fileName)).filter((d) =>
-      this.isErrorDiagnostic(d)
-    );
+    const diagnostics = service.getDiagnostics(fileName).filter((d) => this.isErrorDiagnostic(d));
```

Comment:
...but was unnecessary added to Glit.report, so we've reported less errors than was available.